spec.bs

```diff
@@ -724,6 +791,8 @@ fields:
     :: The instrument information that was displayed to the user.
 </dl>
 
+Q: We should probably state that these fields must be displayed to the user.
```

Comment:
See the big TODO in https://w3c.github.io/secure-payment-confirmation/#sctn-transaction-confirmation-ux ;)

Reply:
Saw it. :-)